crates/router/src/connector/adyen/transformers.rs

```diff
@@ -2751,10 +2751,7 @@ fn get_redirect_extra_details(
                     country,
                     preferred_language,
                     ..
-                } => Ok((
-                    Some(preferred_language.to_string()),
-                    Some(country.to_owned()),
-                )),
+                } => Ok((preferred_language.clone(), country.clone())),
```

Comment:
is this not mandatory now?? earlier it was a mandatory field ryt

Reply:
No country and preferred_language is not a [required field](https://docs.adyen.com/payment-methods/sofort/api-only/#make-payment)